Add documentation for daily forecast card feature (#44865)
home-assistant/home-assistant.github.io: source/dashboards/features.markdown

  type: string
{% endconfiguration %}

## Daily forecast

> Widget that displays a bar chart of daily forecast temperature ranges for a [weather](/integrations/weather) entity. Each bar shows the low-to-high temperature range for the day, colored by the forecasted weather condition, with a thin line across the bars marking the current temperature.

Reviewer: ```suggestion
Widget that displays a bar chart of daily forecast temperature ranges for a weather entity. Each bar shows the low-to-high temperature range for the day, colored by the forecasted weather condition. A thin horizontal line marks the current temperature across the bars.
```

<p class='img'>
  <img src='/images/dashboards/features/daily_forecast.png' alt='Screenshot of the tile card with the daily forecast feature'>
  Screenshot of the tile card with the daily forecast feature
</p>

```yaml
features:
  - type: "daily-forecast"

>     days_to_show: 7

Reviewer: ```suggestion
    days_to_show: 5
```
I would show something other than the default value.

```

{% configuration features %}
type:
  required: true
  description: "`daily-forecast`"
  type: string
forecast_type:
  required: false
  description: "Which forecast feed to use. One of `daily` or `twice_daily`. Only types supported by the weather entity are selectable. Defaults to `daily` when available, otherwise `twice_daily`."
  type: string
days_to_show:
  required: false

>   description: Number of days of forecast data to show. Minimum is 1 day. The available data depends on how far ahead your weather integration provides daily forecasts.

Reviewer: ```suggestion
  description: Number of days of forecast data to show. Minimum is 1 day (showing the forecast for the current day). The available data depends on how far ahead your weather integration provides daily forecasts.
```
something like this maybe?

>   description: Number of days of forecast data to show. Minimum is 1 day. The available data depends on how far ahead your weather integration provides daily forecasts.

Reviewer: ```suggestion
  description: Number of days of forecast data to show. Minimum is 1 day ahead. The available data depends on how far ahead your weather integration provides daily forecasts.
```

Author: It's not really ahead because the first day is the the current one

Reviewer: oh, right. Thx.
